File: array_and_hashing/python/twoSum.py
def twoSum(self, nums: List[int], target: int) -> List[int]:
    dict = {} # key: n | value: index
    for i, n in enumerate(nums):
        compliment = target - n
        if compliment in dict:
            return [i, dict[compliment]]
        dict[n] = i

File: array_and_hashing/swift/twoSum.swift
func twoSum(_ nums: [Int], _ target: Int) -> [Int] {
    var dict = [Int: Int]()
    var compliment : Int = 0
    for (i, n) in nums.enumerated(){
        compliment = target - n
        if dict.keys.contains(compliment){
            return [i, dict[compliment]!]
        }
        dict[n] = i
    }
    return []
}